src/DataValues/DecimalValue.php

```diff
@@ -355,10 +355,11 @@ public function getArrayValue() {
 	 *
 	 * @deprecated since 0.8.3. Static DataValue::newFromArray constructors like this are
 	 *  underspecified (not in the DataValue interface), and misleadingly named (should be named
-	 *  newFromArrayValue). Instead, use DataValue builder callbacks in @see DataValueDeserializer.
+	 *  newFromArrayValue). Instead, use the constructor or use DataValue builder
+	 *  callbacks. (e.g. @see DataValueDeserializer)
```

Comment:
Use of DataValueDeserializers (builders) is one alternative to this deprecated method but this component can be used on it's own. The alternative w/o use of separate component is just construct the object directly